cmd/prometheus/main.go

```diff
@@ -224,6 +225,9 @@ func main() {
 	a.Flag("storage.remote.read-max-bytes-in-frame", "Maximum number of bytes in a single frame for streaming remote read response types before marshalling. Note that client might have limit on frame size as well. 1MB as recommended by protobuf by default.").
 		Default("1048576").IntVar(&cfg.web.RemoteReadBytesInFrame)
 
+	a.Flag("storage.exemplars.exemplars-per-series-limit", "Maximum number of exemplars to store in in-memory exemplar storage per series.").
+		Default("10").IntVar(&cfg.ExemplarsLimit)
```

Comment:
if we assume about 20% of series have exemplars (and have filled all 10 of those exemplars in their storage) then this would result in about a 10% memory increase based on the math in the design doc